src/entry.rs

```diff
@@ -5,9 +5,13 @@ use std::fs::OpenOptions;
 use std::io::prelude::*;
 use std::io::{self, Error, ErrorKind, SeekFrom};
 use std::marker;
+#[cfg(windows)]
+use std::os::windows::fs::OpenOptionsExt;
 use std::path::{Component, Path, PathBuf};
 
 use filetime::{self, FileTime};
+#[cfg(windows)]
```

Comment:
Instead of `#[cfg]` imports, can these imports be placed in the block that's already `#[cfg]` windows below?

Reply:
Done